---
title: "Metagenomics"
image: nerc-metagenomics-card-narrow.png
description: Learn more about metagenomics, a way to explore all the genetic material in an environmental sample.
---
::: {.callout-note appearance="tip"}

## This course is now fully booked - to be added to the waiting list please complete the [online form](https://forms.gle/WZCDADQcisbjmVpX9) 

:::

Metagenomics explores all the genetic material in an environmental sample. It can be used to characterise the taxonomic characteristics of microbial communities.

This online module will be delivered over a three-week period from 13 February - 1 March 2024.

Metagenomics analyses involve a lot of data and can take hours to days to complete! But don't worry! The format of the workshop takes account of this. For longer analyses there will be scheduled online zoom sessions to cover concepts and get started followed by offline time for the analysis run and for you to complete some exercises. These will be supported by online drop-ins and a Slack channel for trouble-shooting.

# Target audience

UK based Environmental Science researchers at any career stage or anyone with an interest in Metagenomics. We assume no prior experience of the command line or high performance computing. 

Check out testimonials from previous attendees!

> "This course was fantastic in both helping me to understand the concepts behind metagenomic polishing/assembly processes and in actually learning how to do it ourselves. I have learnt a lot that I can confidently go and apply to my own research."

> "Really recommend this course! The content was great, covering command line fundamentals and the whole metagenomics workflow in a way that was really easy to understand. The team were super engaged and helpful throughout!"

> "Really pleased with how the course was ran. The instructors were clear and patient, and the online course materials were thorough. I really enjoyed the taxonomic analysis as it brought everything we learned together. This course is super applicable for when I collect and analyse my data further into my PhD project. :)"

# Registration

::: {.callout-note appearance="default"}

## This course is now fully booked - to be added to the waiting list please complete the [online form](https://forms.gle/WZCDADQcisbjmVpX9) 

:::

There are 30 places available and this workshop is free of charge.


# Programme

We start by teaching the essential tools used in High Performance computing such as file systems and the command-line to connect to and use cloud computing for file navigation and script writing. We then introduce metagenomics and building a metagenomic assembly.  During the sessions a soil microbiome example is used however all principles are relevant to any metagenomic analysis. In the second week, you will learn how to improve your assembly by 'polishing', separate your assembled metagenome into individual genomes (MAGS) and conduct taxonomic assignment and analysis. But if you can't get something to work, we have a Slack channel and a weekly drop-in to help.

*Happy learning!*

**Week 1**

|                     |               |                                                                |
|------------------|------------------|------------------------------------|
| Tuesday 13 February  | 9:30 - 11:30 | Command-line programming: file systems, files and directories  |
| Wednesday 14 February | 9:30 - 11:30 | Command-line programming: using the command line               |
| Friday 16 February | 9:30 - 13:00 | An introduction to metagenomics, quality control and assembly. |
|                     |               | Offline time for assembly to complete.                         |

**Week 2**

|                       |               |                                                   |
|------------------|------------------|------------------------------------|
| Monday 19 February    | 13:00 - 14:00 | Online troubleshooting drop-in session (optional) |
| Wednesday 21 February | 13:00 - 14:30 | Polishing your metagenome assembly.               |
|                       |               | Offline time for the polishing to complete        |
| Friday 23 February    | 14:00 - 15:00 | Online troubleshooting drop-in session (optional) |

**Week 3**

|                     |               |                                                              |
|------------------|------------------|------------------------------------|
| Tuesday 27 February | 9:30 - 11:00 | Binning a metagenome assembly into individual genomes (MAGS) |
|                     |               | Offline time for the binning to complete                     |
| Thursday 29 February| 9:30 - 10:30 | Online troubleshooting drop-in session (optional)            |
| Friday 1 March  | 9:30 - 12:30 | Taxonomic Assignment and Analysis                            |

If you are unable to attend an online session, recordings will be distributed. You will have access to your instance until Friday 8 March (one week after the course is completed) to give you time to consolidate your learning. Usage of the instances will be monitored; instances which are inactive will be closed down as a daily charge is incurred.



# Pre-requisites

You will need familiarity with biological concepts, including the concept of microbiome. We assume no prior experience of the command line or high performance computing. Windows users will need to install GitBash.

You don't need to worry about installing metagenomics software or putting the data on your own computer! You will have access to an Amazon Web Services instance with all the data and software and will only need to log in to it.

# Learning outcomes

Following completion of this course, learners will be able to:

-   explain the hierarchical structure of a file system and describe the files and file structure used in the course
-   explain what is meant by a working directory, a path and a relative path and write down paths that they will need for the course
-   start a Terminal (Mac) or Git Bash Terminal (Windows)
-   navigate a file system using the command line
-   log in to and exit their AWS instance (the cloud)
-   use common commands such as ls, pwd and cd, on the command line
-   know the difference between genomics and metagenomics
-   describe the steps in a metagenomic workflow
-   perform quality control on reads and assemble them into a metagenome
-   perform polishing to improve an assembly
-   use binning to separate the metagenome into different species or MAGs (Metagenome-Assembled Genomes)
-   use Kraken 2 to assign taxonomy to reads and contigs and phyloseq in R to analyse taxonomic diversity

# Scholarships and additional support

We offer scholarships to enable members of underrepresented groups and those with financial difficulties to participate in our training courses.

The scholarship scheme can cover approximately Â£150 per application, for costs such as:

- childcare
- accessibility needs
- other costs that might otherwise prevent participation
- second monitors and headsets.

To request a scholarship please indicate this in the registration form, before 12 noon on 1st February 2024. Submissions after this date will not be considered. We will contact you 2-3 days after the deadline to confirm whether you have been awarded a scholarship.
